extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -71,6 +72,19 @@ export class AmpAdUIHandler {
       return;
     }
 
+    const flyingCarpetElements =
```

Comment:
nit: use `getAdContainer` helper method instead

Reply:
Thank you! Honestly, I had tried that before, and I saw that it returned an element. And was confused, so I assumed it didn't work.

Tried again and it did. Was an error on my part, thanks for brining this up! 😄